rules/rbe_repo.bzl

```diff
@@ -696,6 +699,15 @@ _rbe_autoconfig = repository_rule(
                    "example, [\"@bazel_tools//platforms:linux\"]. Default " +
                    " is set to values for rbe-ubuntu16-04 container."),
         ),
+        "exec_properties": attr.string_dict(
+            doc = (
+                "Optional. The execution properties to be used when creating the " +
+                "underlying platform. When providing this attribute, " +
+                "use_legacy_platform_definition must be set to False. Note that " +
+                "the container image property must not be specified via this " +
```

Comment:
container_image.
(will this fail if its specified?)

Reply:
Yes. I call fail(...).
I prefer to do that so that we won't have to deal with breaking people who choose to use it anyway.
If we find that people want to specify container_image using this mechanism, we can open it up in the future.

Reply:
I was actually debating whether to write container-image, container_image or container image.
container-image is the name of the key in the dict.
container_image is the parameter of create_exec_properties_dict.
To avoid confusion, I think I prefer to stick with container image.